src/policy/policy.h

```diff
@@ -21,7 +21,13 @@ class CScript;
 
 /** Default for -blockmaxweight, which controls the range of block weights the mining code will create **/
 static constexpr unsigned int DEFAULT_BLOCK_MAX_WEIGHT{MAX_BLOCK_WEIGHT};
+/** Default for -blockreservedweight **/
 static constexpr unsigned int DEFAULT_BLOCK_RESERVED_WEIGHT{8000};
+/** This accounts for the block header, var_int encoding of the transaction count and a minimally viable
+ * coinbase transaction. It adds an additional safety margin, because even with a thorough understanding
+ * of block serialization, it's easy to make a costly mistake when trying to squeeze every last byte.
+ * Setting a lower value is prevented at startup. */
+static constexpr unsigned int MINIMUM_BLOCK_RESERVED_WEIGHT{2000};
```

Comment:
nit: maybe all of these should live in miner.h?

Reply:
I guess it's a philosophical question. There's mempool policy and there's the defaults we set for miners, which typically match. But the `*_WEIGHT` constants here have no effect on the mempool, so are they policy?

But there's another consideration: `miner.{h,cpp}` consists mostly of utility code, and so doesn't seem like a good place to put these important constants. We could introduce `mining_defaults.h`, but it seems simpler to keep them here.

Reply:
Meh good point miner policy is policy, resolving.